Geometry/ForwardCommonData/python/hfnoseV15ParametersInitialization_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.HGCalCommonData.hgcalEEV15ParametersInitialization_cfi import *
+
+hfnoseParametersInitialize = hgcalEEParametersInitialize.clone(
+    name  = cms.string("HGCalHFNoseSensitive"),
+    name2 = cms.string("HFNoseEE"),
+    nameW = cms.string("HFNoseWafer"),
+    nameC = cms.string("HFNoseCell"),
+    nameT = cms.string("HFNose"),
+    nameX = cms.string("HGCalHFNoseSensitive"),
```

Comment:
```suggestion
    name  = "HGCalHFNoseSensitive",
    name2 = "HFNoseEE",
    nameW = "HFNoseWafer",
    nameC = "HFNoseCell",
    nameT = "HFNose",
    nameX = "HGCalHFNoseSensitive"
```
(safer syntax, as already in Geometry/HGCalCommonData/python/hgcalV15ParametersInitialization_cfi.py